pkg/apis/apps/register.go

```diff
@@ -17,13 +17,15 @@ limitations under the License.
 package apps
 
 import (
+	"k8s.io/apimachinery/pkg/apimachinery/announced"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/kubernetes/pkg/api"
 )
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
+	AddToScheme   = announced.CombineSchemeBuilders(SchemeBuilder.AddToScheme, api.AddToScheme).AddToScheme
```

Comment:
This was neat.  "apps" includes types from the core API, but did not include those types for itself and did not include conversions.  This meant that if you built a scheme with just apps, you'd be unable to roundtrip your types properly.